<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration_form</class>
 <widget class="QWidget" name="registration_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="password_input">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
